server/src/test/java/org/opensearch/extensions/ExtensionsManagerTests.java

```diff
@@ -93,6 +93,9 @@ public class ExtensionsManagerTests extends OpenSearchTestCase {
     private Setting customSetting = Setting.simpleString("custom_extension_setting", "none", Property.ExtensionScope);
     private NodeClient client;
     private MockNioTransport transport;
+    private IdentityService identityService;
+    private Path extensionDir;
```

Comment:
This is not used, right?

Reply:
Oops good catch. Not sure why Precommit did not find that.